Source/WebCore/css/ComputedStyleExtractor.cpp

```diff
@@ -645,32 +642,42 @@ static Ref<CSSValueList> borderRadiusShorthandValue(const RenderStyle& style)
     bool showVerticalTopRight = showVerticalBottomRight || (style.borderTopRightRadius().height != style.borderTopLeftRadius().height);
 
     auto topLeftRadius = borderRadiusCornerValues(style.borderTopLeftRadius(), style);
+    CSSValue* topLeftRadiusX = topLeftRadius->first();
```

Comment:
I would have used `auto` or `auto*`.

Reply:
But then it becomes a `CSSPrimitiveValue*`, which somehow causes a compile error:

```
In file included from /app/webkit/Source/WebCore/css/CSSValuePool.h:28,
                 from /app/webkit/Source/WebCore/css/CSSValuePool.cpp:27,
                 from /app/webkit/WebKitBuild/Release/WebCore/DerivedSources/unified-sources/UnifiedSource-2f84417a-11.cpp:2:
/app/webkit/Source/WebCore/css/CSSPrimitiveValue.h:192:33: error: inline function ‘WebCore::CSSPrimitiveValue::operator T() const [with T = WTF::Ref<WebCore::CSSValue>]’ used but never defined [-Werror]
  192 |     template<typename T> inline operator T() const; // Defined in CSSPrimitiveValueMappings.h
      | 
```

Using `CSSValue*` it worked :shrug: